src/Client.ts

```diff
@@ -165,7 +172,7 @@ export class Client extends Eris.Client implements ClientOptions {
 		}
 		// Separate command name from arguments and find command object
 		const args = content.split(' ');
-		const commandName = args.shift();
+		const commandName = this.caseSensitiveCommand ? args.shift() : args.shift().toLowerCase();
```

Comment:
I am not sure how this line works exactly because the next line checks if commandName is undefined. This is implying to me that args.shift() can possibly be undefined. Therefore .toLowerCase() on undefined could possibly throw an error. Could you double-check this?

Reply:
You see the line is checking if `Client#caseSensitiveCommand` is true. If it is, then it does not minuscules the name of the command.

Reply:
This is the error @Skillz4Killz is referring to:
![Error: Object is possibly 'undefined'.](https://user-images.githubusercontent.com/4165301/71315222-bc507200-2425-11ea-9748-8b8b40a01cc2.png)

I'll fix this in an additional commit.